src/Compilers/CSharp/Portable/CSharpResources.resx

```diff
@@ -4390,10 +4390,13 @@ To remove the warning, you can use /reference instead (set the Embed Interop Typ
 
                         - CODE GENERATION -
  /debug[+|-]                   Emit debugging information
- /debug:{full|pdbonly}         Specify debugging type ('full' is default, and 
+ /debug:{full|pdbonly|portable}
+                               Specify debugging type ('full' is default, and 
                                enables attaching a debugger to a running 
-                               program)
+                               program. 'portable' is a cross-platform format)
  /optimize[+|-]                Enable optimizations (Short form: /o)
+ /deterministic                Produce a deterministic assembly
```

Comment:
I'd rather have further explanation outside the help text. Once you've understood what the flag means you won't be helped by always seeing "bitwise". Before you've understood what it means that isn't really enough of a hint. This help text is just supposed to be a reminder of the spelling of flags that you intend to use, not an explanation of them. I'm working on a longer explanatory document, but it won't be shipped in the box.